ta/concurrent/Android.mk

```diff
@@ -0,0 +1,7 @@
+include $(CLEAR_VARS)
```

Comment:
I understood they should be named `Android.mk`, but something like `ta_android.mk` Am I wrong.